src/allmydata/scripts/create_node.py

```diff
@@ -344,6 +348,74 @@ def create_node(config):
         os.mkdir(basedir)
     write_tac(basedir, "client")
 
+    # if we're doing magic-wormhole stuff, do it now
+    if config['join'] is not None:
+        print >>out, "Opening wormhole with code '{}'".format(config['join'])
+        relay_url = config.parent['wormhole-server']
+        print >>out, "Connecting to '{}'".format(relay_url)
+
+        wh = wormhole.create(
+            appid=u"tahoe-lafs.org/lafs",
```

Comment:
>Did we release anything with the "v1" appid in it?

Nope.. I remember @exarkun suggesting earlier, however, that a version-bump might make for good practice (but don't recall us reaching any sort of consensus on moving forward with that idea -- and may even be mistaken in thinking that he was referring to the `appid` specifically now that I think about it..). Personally, I have no qualms either way, but wanted to raise the the question here in the event that anyone felt strongly about what the `appid` should be more generally (since it isn't something we can easily change if/when this PR lands); like you implied, what really matters here is that everything is compatible.

>Also, is the appid the right place to do versioning like this, or in the capabilities exchanged as the first messages?

Hmm.. That's a good question. Reducing the possibility of "crossing the streams" by using separate `appid`s where possible is desirable for both security and usability reasons (since an attacker would need to flood channels on _both_ `appid`s instead of just one -- while fewer nameplates in use means potentially fewer characters to type!) and I was already operating under the assumption that we would be doing this later on to isolate magic-folder-invites from grid-invites (or at least I can't really think of any _good_ reasons right now for keeping the `appid`s the same for both types). On that note, I was under the impression -- and please correct me if I'm wrong -- that the capabilities-exchange was intended more so as a means of handling the situation in which we might want/need to change the data-format of a specific invite-type in a way that is not backwards-compatible with the previous one (i.e., as a means of signaling to users more broadly that they need to update their clients) -- in other words, a means of migrating from grid-invite "version 1" to grid-invite "version 2" independently of the `appid` (and not so much as a means of packing magic-folder-invites in with grid-invites under the same `appid`).

All that being said, I suppose it doesn't _really_ matter at this point, since we're only handling one "type" of invite right now and the capabilities-exchange gives us a good way -- better than changing the `appid`, certainly -- of updating its data-format in the unlikely event that we ever need to (and in retrospect I probably should have framed my original comment to more clearly distinguish between a) the question of isolating future invite-"types" and b) the question of versioning since they can be addressed independently of one another).

Maybe, then, we should just omit the "v1" suffix from the current `appid` altogether (i.e., use `tahoe-lafs.org/tahoe-lafs` _everywhere_) and use the capabilities-exchange to handle any/all "versioning"/update-messaging stuff moving forward? Using the `appid` for versioning seems like a pretty bad idea to me overall (since, in practice, a sender would always have to open at least _two_ wormholes -- one using the "old" `appid`(s) and one using the "new" -- in order to tell clients on the old one to update).. What do you think?